lib/Auth/Process/PerunEntitlement.php

```diff
@@ -114,6 +114,12 @@ private function getEduPersonEntitlement(&$request)
 
         $eduPersonEntitlement = [];
         foreach ($groups as $group) {
+            $groupName = $group->getUniqueName();
+
+            if (preg_match('/^\w*\:members$/', $groupName)) {
+                $groupName = preg_replace('/^(\w*)\:members$/', '$1', $groupName);
+            }
```

Comment:
The `if` is not needed, just `$groupName = preg_replace('/^(\w*)\:members$/', '$1', $groupName);` does the same thing.

Reply:
Resolved